From: Iain Lane <iainl@gnome.org>
Date: Wed, 6 Mar 2019 12:33:33 +0000
Subject: build: Add mutter's private directory to libst's build-time RPATH

We need this to run `test-theme`, otherwise when run as part of the
build tests it fails like:

  error while loading shared libraries: libmutter-cogl-4.so.0: cannot
  open shared object file: No such file or directory

Origin: upstream, commit:c7342824
---
 src/st/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/st/meson.build b/src/st/meson.build
index 38ae5c7..444ed49 100644
--- a/src/st/meson.build
+++ b/src/st/meson.build
@@ -126,6 +126,7 @@ libst = shared_library('st-1.0',
   sources: st_gir_sources + st_non_gir_sources,
   c_args: st_cflags,
   dependencies: [clutter_dep, gtk_dep, croco_dep, x11_dep, m_dep],
+  build_rpath: mutter_typelibdir,
   install_rpath: mutter_typelibdir,
   install_dir: pkglibdir,
   install: true
